extensions/barrage/src/main/java/io/deephaven/extensions/barrage/chunk/BooleanChunkReader.java

```diff
@@ -138,4 +138,14 @@ private static void useValidityBuffer(
             chunk.fillWithNullValue(offset + ei, pendingSkips);
         }
     }
+
+    /**
+     * Returns the number of longs needed to represent a single bit per element.
+     *
+     * @param numElements the number of rows
+     * @return number of longs needed to represent numElements bits rounded up to the nearest long
+     */
+    private static int getNumLongsForBitPackOfSize(final int numElements) {
```

Comment:
Moved because BaseChunkInputStreamGenerator and subtypes can't be compiled to JS, so this method isn't accessible, should not keep multiple copies of it long-term.

Reply:
Will revert, Ryan's workaround makes it so I don't need this any more.